spec/integration/opi_apps_client_spec.rb

```diff
@@ -43,12 +43,20 @@ def up?(url)
   context 'OPI system tests' do
     context 'Desire an app' do
       let(:droplet) {
-        VCAP::CloudController::DropletModel.new(
+        VCAP::CloudController::DropletModel.make(
           docker_receipt_image: 'http://example.org/image1234',
           droplet_hash: 'd_haash',
           guid: 'some-droplet-guid'
         )
       }
+      let(:lifecycle_type) { nil }
+      let(:app_model) {
```

Comment:
`app_model` is never referenced, and therefore not instantiated. How were you expecting to use it?